adapters/coap-vertx-base/src/main/java/org/eclipse/hono/adapter/coap/DefaultDeviceResolver.java

```diff
@@ -111,12 +131,15 @@ public AdditionalInfo getInfo(final Principal clientIdentity) {
         final Map<String, Object> result = new HashMap<>();
 
         if (clientIdentity instanceof PreSharedKeyIdentity) {
+            final Span span = newSpan("PSK-INFO");
```

Comment:
IMHO the operation should be a little more expressive, e.g. *get device identity info* or something like that?

Reply:
Adapted to "PSK-getDeviceIdentityInfo"